### DISCUSSION TITLE
Nvidia omniverse 使用體驗

### DISCUSSION THREAD

**Discussion by 4rfvedc** [1101-GS4719-C/Python-Community-GS4719-C#1472] [General]
NVIDIA Omniverse是一個複雜的擴展平台，主要用於多人對3d場景的設計與開發，並提供實時光追渲染器與各式物理模擬引擎，包括剛體，布料與流體等，其透過gpu加速的physx物理引擎可以比傳統引擎快上百倍，非常適合用於AI於特定用途的訓練。omniverse同時也支援自然語言模型與Audio2Face等等AI技術，許多公司已經開始使用該軟體於工業化中，並有效加速了開發過程，omniverse是當前元宇宙開發的重要基礎建設。

本人主要使用physx引擎進行訓練基於物理的人形角色，透過該引擎高效的採樣效率，對整個訓練過程進行了非常優秀的加速，animation retargeting功能簡化了動補數據骨架轉移的時間，而功能強大的real time renderer也讓可視化結果方便了許多。

https://docs.omniverse.nvidia.com/plat_omniverse/plat_omniverse/overview.html

**Comment by KozakHou** [comment]
我用的是Modulus應用在Omniverse上，就如同你上面寫的Physx物理引擎，不過Modulus是要自己"訓練"自己的物理引擎，像是熱流、風向、彈性阻尼項等。其實比較像是客製化的物理宏觀世界。
這類的搭配最有名的例子就是NVIDIA的Earth-2模型，用來模擬地球天氣系統，可以推廣到幾天、幾個禮拜到幾年後的地球受極端氣候的影，蠻屌的。

https://blogs.nvidia.com.tw/2021/11/12/earth-2-supercomputer/
